src/meta/src/lib.rs

```diff
@@ -65,7 +67,10 @@ enum Backend {
 pub struct MetaNodeOpts {
     // TODO: rename to listen_address and separate out the port.
     #[clap(long, default_value = "127.0.0.1:5690")]
-    host: String,
+    listen_addr: String,
```

Comment:
Plz fix the opt in risedev as also. https://github.com/singularity-data/risingwave/blob/ce23e4a099867e10f2426eefc00d7475175b1e33/src/risedevtool/src/task/meta_node_service.rs#L45

Reply:
ok.